engines/glk/adrift/detection_tables.h

```diff
@@ -382,7 +492,10 @@ const GlkDetectionEntry ADRIFT_GAMES[] = {
 	DT_ENTRY0("murdererhadleft", "f95c63f90f28061a15dbe3bdf0cd4048", 13425),
 	DT_ENTRY0("mustescape", "e58fd08918fa3d5a38f424f79595fb4e", 17497),
 	DT_ENTRY0("mymindsmishmash", "593f536f38418fdd334a6fa50fd79d97", 68353),
+	DT_ENTRY0("mysteryhouse", "9c7f8577c506737cc07d026d8318598e", 4916),
 	DT_ENTRY0("neighboursfromhell", "686d8c0995aa085fb056c6a5bd402911", 13520),
+	DT_ENTRY0("oct31", "7ca7980f7adf3da370464610b702e94b", 356192),
+	DT_ENTRY1("oct31", "Danish", "7f0637944bc69e2bff843e76689b645b", 194758), // TODO: How can I enter Danish language?
```

Comment:
Like this:
`DT_ENTRYL0("oct31", Common::DA_DNK, "7f0637944bc69e2bff843e76689b645b", 194758),`

Reply:
Fixed the entry